### DISCUSSION TITLE
Markdown tables in <TabItem></TabItem>

### DISCUSSION THREAD

**Discussion by ErikQQY** [facebook/docusaurus#4761] [Q&A] [answered]
Hello, I am migrating docusaurus v1 website to v2, I want to know how can we put a table in Tabs.

For example,
[![gJf2VS.png](https://z3.ax1x.com/2021/05/09/gJf2VS.png)](https://imgtu.com/i/gJf2VS)

In v1 docusaurus, we can successfully put a table in a tab.

However, in v2 docusaurus, it doesn't work.
[![gJfTK0.png](https://z3.ax1x.com/2021/05/09/gJfTK0.png)](https://imgtu.com/i/gJfTK0)

**Comment by slorber** [comment, marked as answer]
Hi,

You can interleave markdown inside MDX JSX elements (`<TabItem>`) by using line breaks:

```
<Tabs
  defaultValue="apple"
  values={[
    {label: 'Apple', value: 'apple'},
    {label: 'Orange', value: 'orange'},
    {label: 'Banana', value: 'banana'},
  ]}>
  <TabItem value="apple">

  | foo | bar |
  | :-- | :-: |
  | baz | qux |

  </TabItem>
  <TabItem value="orange">This is an orange 🍊</TabItem>
  <TabItem value="banana">This is a banana 🍌</TabItem>
</Tabs>
```

![image](https://user-images.githubusercontent.com/749374/117790150-5d251480-b249-11eb-8961-05de592237ab.png)

**Reply by slorber** [reply]
Note, this is something coming with our MDX parser, so all the docs of MDX can be helpful to solve the edge cases

Also, the MDX playground is useful to debug either something is a bug of Docusaurus or MDX: https://mdxjs.com/playground/

**Reply by ErikQQY** [reply]
I think I should first study more about MDX. Thanks a lot!!!!!
